tests/test_watchman.py

```diff
@@ -126,7 +126,7 @@ def test_empty_scopes(self):
         with self.assertRaises(AttributeError):
             getattr(Environment, 'scopes')
             getattr(Environment, 'rate_limit')
-            print getattr(Environment, 'decorators')
+            print(getattr(Environment, 'decorators'))
```

Comment:
seems like this print statement lingered from debugging, might want to just keep it as a `getattr`